Sources/SendGrid/Provider.swift

```diff
@@ -1,6 +1,6 @@
 import Vapor
 
-public final class Provider: Vapor.Provider {
+public final class SendGridProvider: Vapor.Provider {
```

Comment:
I would rename the SendGrid class to SendGridClient or SendGridMail instead.
Both are breaking changes either way.